README.md

```diff
@@ -114,14 +148,32 @@ any given point in time, fully support a maximum of 2 LTS releases.
 [ICal]: schedule.ical
 [`NAN`]: https://github.com/nodejs/nan
 
-## LTS Team members
+The working group members are the union of the LTS, Releasers
+and CITGM team members listed below.
 
+## LTS Team members
 * Gibson Fahnestock [@gibfahn](https://github.com/gibfahn)
 * James M Snell [@jasnell](https://github.com/jasnell)
 * Jeremiah Senkpiel [@Fishrock123](https://github.com/Fishrock123)
 * Michael Dawson [@mhdawson](https://github.com/mhdawson)
 * Myles Borins [@MylesBorins](https://github.com/MylesBorins)
-* Rod Vagg [@rvagg](https://github.com/rvagg)
 * Sam Roberts [@sam-github](https://github.com/sam-github)
 
-Github team for LTS: https://github.com/orgs/nodejs/teams/lts
+### Releasers team
+* Colin Ihrig [@cjhrig](https://github.com/cjihrig)
+* Evan Lucas [@evanlucas](https://github.com/evanlucas)
+* Italo A. Casas [@italoacasas](https://github.com/italoacasas)
+* James M Snell [@jasnell](https://github.com/jasnell)
+* Jeremiah Senkpiel [@Fishrock123](https://github.com/Fishrock123)
+* Myles Borins [@MylesBorins](https://github.com/MylesBorins)
+* Rod Vagg [@rvagg](https://github.com/rvagg)
+
+### CITGM team
+* James M Snell [@jasnell](https://github.com/jasnell)
+* Myles Borins [@MylesBorins](https://github.com/MylesBorins)
+* Gibson Fahnestock [@gibfahn](https://github.com/gibfahn)
+* George Adams [George Adams](https://github.com/gdams)
```

Comment:
yeah so it should be `[gdams](https://github.com/gdams)` instead of `[George Adams](https://github.com/gdams)`